Travel_fellowships.md

```diff
@@ -92,6 +92,12 @@ However, this fellowship may cover the rental cost of the large hardware (for ex
 The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/or microphone.
 If the cap is insufficient, applicants can describe their hardware needs in their application.
 
+### Expenses and Cancellation
+
+We recommend you purchase travel insurance, and include this as part of your expenses claim.
+In the event of cancellation, we would expect to reimburse your travel insurance premium and any insurance excess fee.
```

Comment:
The premium is what the person pays regardless, right? 

The excess can be much higher than the premium, and might be over the $1000 cap. We may want to say something like "as long as the total reimbursement does not exceed the $1000 award."

Reply:
True - although I don't recall seeing an excess that high personally.